resources/function_help/json/substr_count

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "substr_count",
+  "type": "function",
+  "groups": [
+    "String"
+  ],
+  "description": "Counts the number of occurrences of a substring within a string. By default, the function counts non-overlapping occurrences (like in Python). If the third argument is set to true, the function counts overlapping occurrences (like in QT).",
+  "arguments": [
+    {
+      "arg": "input",
+      "description": "The input string to search in."
+    },
+    {
+      "arg": "substring",
+      "description": "The substring to search for."
+    },
+    {
+      "arg": "overlapping",
+      "description": "Optional flag to control whether overlapping occurrences should be counted. Defaults to false (non-overlapping counting). Set to true to count overlapping occurrences."
+    }
+  ],
+  "examples": [
+    {
+      "expression": "substr_count('banana', 'an')",
+      "returns": "2",
+      "description": "Counts non-overlapping occurrences of 'an' in 'banana'."
+    },
+    {
+      "expression": "substr_count('Funniness', 'n')",
+      "returns": "3",
+      "description": "Counts non-overlapping occurrences of 'n' in 'Funniness'."
+    },
+    {
+      "expression": "substr_count('aaaaa', 'aa')",
+      "returns": "4",
+      "description": "Counts non-overlapping occurrences of 'aa' in 'aaaaa'."
+    },
+    {
+      "expression": "substr_count('aaaaa', 'aa', true)",
+      "returns": "4",
+      "description": "Counts overlapping occurrences of 'aa' in 'aaaaa'. The substring 'aa' appears four times in an overlapping fashion."
```

Comment:
Is "fashion" the right word here? I'm not a native English speaker. It looks like the word "fashion" is not used in any other function description.

Reply:
I can maybe write `as it overlaps`? Does it sounds better for you? Thanks

Reply:
Actually, you are right: "fashion" should be OK (I asked because that word is usually used in Italian with a different meaning). I would leave it as is. Thanks.